includes/Core/REST_API/REST_Routes.php

```diff
@@ -603,99 +270,14 @@ function ( $response ) {
-	 * @return array Module REST response data.
-	 */
-	private function prepare_module_data_for_response( Module $module ) {
-		$manager = $this->modules;
-
-		return array(
-			'slug'         => $module->slug,
-			'name'         => $module->name,
-			'description'  => $module->description,
-			'homepage'     => $module->homepage,
-			'internal'     => $module->internal,
-			'active'       => $manager->is_module_active( $module->slug ),
-			'connected'    => $manager->is_module_connected( $module->slug ),
-			'dependencies' => $manager->get_module_dependencies( $module->slug ),
-			'dependants'   => $manager->get_module_dependants( $module->slug ),
-		);
-	}
-
-	/**
-	 * Gets the REST schema for a module.
-	 *
-	 * @since 1.0.0
-	 *
-	 * @return array Module REST schema.
-	 */
-	private function get_module_schema() {
-		return array(
-			'$schema'    => 'http://json-schema.org/draft-04/schema#',
-			'title'      => 'module',
-			'type'       => 'object',
-			'properties' => array(
-				'slug'         => array(
-					'type'        => 'string',
-					'description' => __( 'Identifier for the module.', 'google-site-kit' ),
-					'readonly'    => true,
-				),
-				'name'         => array(
-					'type'        => 'string',
-					'description' => __( 'Name of the module.', 'google-site-kit' ),
-					'readonly'    => true,
-				),
-				'description'  => array(
-					'type'        => 'string',
-					'description' => __( 'Description of the module.', 'google-site-kit' ),
-					'readonly'    => true,
-				),
-				'homepage'     => array(
-					'type'        => 'string',
-					'description' => __( 'The module homepage.', 'google-site-kit' ),
-					'format'      => 'uri',
-					'readonly'    => true,
-				),
-				'internal'     => array(
-					'type'        => 'boolean',
-					'description' => __( 'Whether the module is internal, thus without any UI.', 'google-site-kit' ),
-					'readonly'    => true,
-				),
-				'active'       => array(
-					'type'        => 'boolean',
-					'description' => __( 'Whether the module is active.', 'google-site-kit' ),
-				),
-				'connected'    => array(
-					'type'        => 'boolean',
-					'description' => __( 'Whether the module setup has been completed.', 'google-site-kit' ),
-					'readonly'    => true,
-				),
-				'dependencies' => array(
-					'type'        => 'array',
-					'description' => __( 'List of slugs of other modules that the module depends on.', 'google-site-kit' ),
-					'items'       => array(
-						'type' => 'string',
-					),
-					'readonly'    => true,
-				),
-				'dependants'   => array(
-					'type'        => 'array',
-					'description' => __( 'List of slugs of other modules depending on the module.', 'google-site-kit' ),
-					'items'       => array(
-						'type' => 'string',
-					),
-					'readonly'    => true,
-				),
-			),
-		);
+		/**
+		 * Filters the list of available REST routes.
+		 *
+		 * @since n.e.x.t
+		 *
+		 * @param array $routes List of REST_Route objects.
```

Comment:
```suggestion
		 * @param REST_Route[] $routes List of REST_Route objects.
```

Reply:
Not sure on this one, at least not in scope of this issue here. The WP core syntax is typically just the generic `array`. But I see the value of what you're proposing as it provides more specific information. Maybe let's open a separate issue to update such doc blocks for any indexed array.

Reply:
Not sure on this one, at least not in scope of this issue here. The WP core syntax is typically just the generic `array`. But I see the value of what you're proposing as it provides more specific information. Maybe let's open a separate issue to update such doc blocks for any indexed array.